test/unit/error.spec.ts

```diff
@@ -62,3 +62,239 @@ test("re-throw (no catch var)", () => {
     const result = util.transpileAndExecute(source);
     expect(result).toBe("z");
 });
+
+test("return from try", () => {
+    const code = `
+        function foobar() {
+            try {
+                return "foobar";
```

Comment:
Can we get this test duplicated with a void return?

Reply:
That's what's being tested below in `return nil from try`, or am I misunderstanding what you're asking here?

Reply:
ah yes looked over that one.